platform/cog-platform-fdo.c

```diff
@@ -257,10 +257,11 @@ configure_surface_geometry (int32_t width, int32_t height)
 static void
 resize_window (void)
 {
-    wl_egl_window_resize (win_data.egl_window,
-                          win_data.width,
-                          win_data.height,
-                          0, 0);
+    if (win_data.egl_window)
+        wl_egl_window_resize (win_data.egl_window,
+			      win_data.width,
+			      win_data.height,
+			      0, 0);
```

Comment:
Now that you pointed this fix, it *does* seem plausible that a resize event can be triggered during the small time interval before the `.egl_window` member is initialized :scream:

Reply:
I just got word from @zdobersek by chat that this also looks like a good fix to him :+1: